pkg/wallet/wallet.go

```diff
@@ -21,14 +23,16 @@ const (
 	PBKDF2NbRound             = 10000
 	FileModeUserReadWriteOnly = 0o600
 	Base58Version             = 0x00
+	SaltSize                  = 12
```

Comment:
Why 12 now ?

Reply:
to  be iso with massa-client https://github.com/massalabs/massa/blob/main/massa-cipher/src/constants.rs

Reply:
new pr created